src/demo/contexts/demo/defaultImages.ts

```diff
@@ -43,6 +43,18 @@ export const defaultImages: UrlOption[] = [
     label: 'House',
     imageType: 'image',
   },
+  {
+    type: 'url',
+    value: `http://localhost:3000/img/roiTestingImage.png`,
```

Comment:
The image is already publicly available:
https://github.com/image-js/image-dataset-demo/blob/main/docs/particles/spheres.png
https://image-js.github.io/image-dataset-demo/particles/spheres.png